### PR TITLE
Update grpc_client_streaming_client.out

### PR DESCRIPTION
**Pull request by shavidissa**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by daneshk — APPROVED** [review]

File: examples/grpc-client-streaming/grpc_client_streaming_client.out [hunk 1]
```diff
@@ -1,11 +1,11 @@
-# To run the client and send request, create ballerina project and package
-# inside the project and include below files to package directory. Build the
-# project using build-in ballerina tool.
+# Create a Ballerina project and a package inside it. 
+# Add the following files inside the package directory. Build the
+# project using the built-in Ballerina tool.
```
Comment (pubudu91) [inline examples/grpc-client-streaming/grpc_client_streaming_client.out]: The following needs to be rephrased, if this built-in tool referred to here is the build command: "Build the project using the built-in Ballerina tool."

Comment (daneshk) [reply examples/grpc-client-streaming/grpc_client_streaming_client.out]: @shavidissa, shall we make it like "Add the following files inside the package directory and build the project".
Thanks

Comment (samgnaniah) [reply examples/grpc-client-streaming/grpc_client_streaming_client.out]: Please close without merging. Fix can be found in https://github.com/ballerina-platform/ballerina-examples/pull/777

**Comment by daneshk** [comment]
Closing this PR, since this fix is addressed in PR #778

**Pull request closed without merging**